index.html.orig

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
```

Comment:
For whatever reason your push includes 2 html documents.

1. index.html
2. index.html.orig

Reply:
will fix

Reply:
removed it